package.json

```diff
@@ -5,53 +5,52 @@
   "dependencies": {
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
-    "@mui/x-date-pickers": "^6.8.0",
-    "@next/bundle-analyzer": "^13.1.6",
-    "@noble/hashes": "^1.3.0",
+    "@mui/x-date-pickers": "^6.10.1",
+    "@next/bundle-analyzer": "^13.4.12",
+    "@noble/hashes": "^1.3.1",
     "@scure/base": "^1.1.1",
-    "@scure/btc-signer": "^0.5.1",
-    "@sentry/nextjs": "^7.58.1",
-    "@stacks/common": "^6.0.0",
-    "axios": "^1.2.6",
-    "bip32": "^3.1.0",
+    "@scure/btc-signer": "^1.0.1",
+    "@sentry/nextjs": "^7.60.0",
+    "@stacks/common": "^6.5.5",
+    "axios": "^1.4.0",
+    "bip32": "^4.0.0",
     "bitcoin-address-validation": "^2.2.1",
-    "bitcoinjs-lib": "github:deezy-inc/bitcoinjs-lib",
-    "clsx": "^1.2.1",
-    "dayjs": "^1.11.8",
+    "bitcoinjs-lib": "^6.1.3",
+    "clsx": "^2.0.0",
+    "dayjs": "^1.11.9",
     "ecpair": "^2.1.0",
     "ethers": "^5.7.2",
-    "framer-motion": "^10.12.16",
+    "framer-motion": "^10.13.1",
     "install": "^0.13.0",
     "lottie-react": "^2.4.0",
     "match-sorter": "^6.3.1",
-    "next": "^13.1.6",
-    "next-themes": "^0.1.1",
+    "next": "^13.4.12",
+    "next-themes": "^0.2.1",
     "nextjs-google-analytics": "^2.3.3",
-    "nosft-core": "^1.3.49",
-    "nostr-tools": "^1.6.0",
-    "npm": "^9.6.5",
+    "nosft-core": "file:../nosft-core",
```

Comment:
this is wrong.

Reply:
Yeah. You said "I will publish the core", I said: After that I can change the version. @andredevjs